MDI_Library/mdi_f90.F90

```diff
@@ -327,6 +327,12 @@ FUNCTION MDI_Conversion_Factor_(in_unit, out_unit, conv) bind(c, name="MDI_Conve
        INTEGER(KIND=C_INT)                      :: MDI_Conversion_Factor_
      END FUNCTION MDI_Conversion_Factor_
 
+     FUNCTION MDI_String_to_Atomic_Number_(atomic_name, atomic_number) bind(c, name="MDI_String_to_Atomic_Number")
```

Comment:
Capitalization in `mdi_f90.F90` typically doesn't matter, because Fortran is case insensitive.  The capitalization of the C name at the end of this line does matter, so it should be `MDI_String_to_atomic_number`.